src/HDF5.jl

```diff
@@ -309,16 +309,16 @@ end
 eltype(::Type{VariableArray{T}}) where T = T
 
 # VLEN objects
-struct HDF5Vlen{T}
+struct Vlen{T}
```

Comment:
Just a thought: should this be `VLen` to be closer to standard camel case?

Reply:
Agreed